molviewspec/molviewspec/builder.py

```diff
@@ -199,6 +210,108 @@ def focus(
         return self
 
 
+class _ClipMixin(_BuilderProtocol):
+    @overload
+    def clip(
+        self,
+        *,
+        type: Literal["plane"],
+        normal: Vec3[float],
+        point: Vec3[float],
+        check_transform: Mat4 | Sequence[float] | None = None,
+        invert: bool = False,
```

Comment:
`invert` seems superfluous here, as one can just flip the normal.
(but if it's here for consistency, it's fine)

Reply:
Yeah, for consistency.